web/controllers/game_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Yggdrasil.GameController do
+  use Yggdrasil.Web, :controller
+
+  alias Yggdrasil.Game
+
+  def index(conn, _params) do
+    games = Repo.all Game
```

Comment:
Good point about eager/lazy loading, I actually think that can be entirely handled in a query according to [this](https://hexdocs.pm/ecto/Ecto.Query.html#preload/3). If you scroll down a bit it mentioned the use directly in a query. 

Ecto seems to be split up nicely, so like for example Sessions in Nhibernate provide the tracking of changes and batching of those changes, where as Ecto relies on changesets and I don't think the Repo keeps track of any of that. Same with querying, the Repo doesn't provide it just executes the query passed in.

So it will be interesting to see how we organize compared to our experiences with Nhibernate. I'm really curious to see and see what they bring in 2.0.

I also completely agree with the use of primitives, that should be restricted to only places where it's needed which hopefully is rare.